src/lib/child_processes/tester.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
```

Comment:
Should have a comment indicating how this script is to be used.

Reply:
:heavy_check_mark: